synth-converter/src/graph/namespaces/cat.rs

```diff
@@ -0,0 +1,31 @@
+use lazy_static::lazy_static;
+use sophia::api::ns::Namespace;
+use sophia_api::namespace;
+namespace! {
+    "http://example.org/cat#",
+    containerID,
```

Comment:
chore: Very nice! So you are using now also typed stuff for the different things like `allores::chemicalName`
This is much better than using `alores::NS::get("chemicalName")` -> why: because mistyping this leads to errors!